src/sentry/models/auditlogentry.py

```diff
@@ -394,8 +394,14 @@ def get_note(self):
             return f"changed on-demand budget to {next_ondemand_budget}"
         elif self.event == AuditLogEntryEvent.TRIAL_STARTED:
             return "started trial"
+
         elif self.event == AuditLogEntryEvent.PLAN_CHANGED:
-            return "changed plan to {}".format(self.data["plan_name"])
+            plan_name = self.data["plan_name"]
+            if "quotas" in self.data:
+                quotas = self.data["quotas"]
+                return f"changed plan to {plan_name} with {quotas}"
```

Comment:
@mbauer404 Out of curiosity, is `quotas` going to be a pre-formatted string?

Reply:
@dashed yes it's a pre-formatted string to avoid additional changes in sentry per data category